modules/oe_content_entity/modules/oe_content_entity_organisation/oe_content_entity_organisation.post_update.php

```diff
@@ -17,3 +17,10 @@ function oe_content_entity_organisation_post_update_00001(): void {
   $storage = new FileStorage(drupal_get_path('module', 'oe_content_entity_organisation') . '/config/post_updates/00001_create_stakeholder_bundle');
   \Drupal::service('config.installer')->installOptionalConfig($storage);
 }
+
+/**
+ * Enable oe_content_organisation_reference module.
+ */
+function oe_content_entity_organisation_post_update_00002() {
+  \Drupal::service('module_installer')->install(['oe_content_organisation_reference']);
```

Comment:
This is not needed, the oe_content_entity_organisation module does not depend from oe_content_organisation_reference